src/Command/User/UserAddCommand.php

```diff
@@ -118,63 +118,94 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         // Process the [email] argument.
-        $email = $input->getArgument('email');
-        if (!$email) {
+        $email = null;
+        if ($emailOrId = $input->getArgument('email')) {
```

Comment:
Shouldn't we set `$email` somewhere in this block? Right now we are sending empty value on invitations

Reply:
Yes, thanks